test/data/halconfig.yml

```diff
@@ -5,7 +5,7 @@ providers:
     enabled: true
     accounts:
     - name: k8s
-      providerVersion: 2
```

Comment:
Not directly related to this change but we can actually remove `providerVersion` since Clouddriver will now default this to `V2`!

Reply:
Good call; for now I've left it I guess as a check that we don't fail if it's present, but I removed it from the expected clouddriver output.